goodpeer/goodpeer.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+from collections import OrderedDict
+from pyln.client import Plugin, Millisatoshi
+
+plugin = Plugin()
+
+
+def get_channel_infos(chan):
+    """
+    Filter out what we need from a channel entry in listchannels
+    """
+    return {
+        "scid": chan["short_channel_id"],
+        "ppm_fee": chan["fee_per_millionth"],
+        "base_fee": chan["base_fee_millisatoshi"],
+        "value": chan["amount_msat"],
+    }
+
+
+def get_nodes(plugin):
+    """
+    Get public Lightning network nodes.
+
+    :return: A dict with the node id as key, and a list of channels as values.
+    """
+    peers = {}
+    channels = plugin.rpc.listchannels()
+    for chan in channels["channels"]:
+        if chan["fee_per_millionth"] == chan["base_fee_millisatoshi"] == 0:
```

Comment:
Interesting, I didn't know that this sort of transitive equality works in python, I'd have expected the resolution to be the following:

```
0 == 0 == 0
True == 0
False
```

But it seems to work when I test it locally. Nice find :+1:

Reply:
Yep it's particularly cool for cases like
```python
if x < y < z:
```